### PR TITLE
Ernazar

### PR DESCRIPTION
**Pull request by Khareken**
added feature add transaction

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,8 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext {
+        kotlin_version = '1.4.10'
+    }
```
Comment (MaratKhakim) [inline build.gradle:5]: No need for this

File: feature-addtransaction/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1 @@
+<manifest package="com.shagalalab.feature.mylibrary"/>
```
Comment (MaratKhakim) [inline feature-addtransaction/src/main/AndroidManifest.xml:1]: You need to use your feature name

**Review by MaratKhakim — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 3]
```diff
@@ -1,11 +1,12 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:${Versions.gradle}"
+        classpath "com.android.tools.build:gradle:4.1.0"
```
Comment (MaratKhakim) [inline build.gradle:9]: Please, don't change it

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
-#Fri Aug 07 20:35:45 AEST 2020
+#Thu Oct 15 03:41:49 UZT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (MaratKhakim) [inline gradle/wrapper/gradle-wrapper.properties:6]: Please, don't change it

**Review by MaratKhakim — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 3 above)
Comment (Khareken) [reply build.gradle:9]: i only deleted

Comment (Khareken) [reply build.gradle:9]: ext {
        kotlin_version = '1.4.10'
    }

Comment (atabekm) [reply build.gradle:9]: Revert all changes in this file, they are not related to your task

File: feature-addtransaction/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -0,0 +1 @@
+<manifest package="com.shagalalab.feature.addtransaction"/>
```
Comment (atabekm) [inline feature-addtransaction/src/main/AndroidManifest.xml:1]: Looks like you haven't read the ticket description carefully, as I've provided there a proper module name

File: gradle/wrapper/gradle-wrapper.properties [hunk 6]
```diff
@@ -1,6 +1,6 @@
-#Fri Aug 07 20:35:45 AEST 2020
+#Fri Oct 16 01:55:17 UZT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```
Comment (atabekm) [inline gradle/wrapper/gradle-wrapper.properties:6]: Revert all changes here

File: settings.gradle [hunk 7]
```diff
@@ -4,6 +4,7 @@ include ':core-prefs'
 include ':core-resources'
 include ':core-usf'
 include ':core-utils'
+include ':feature-addtransaction'
```
Comment (atabekm) [inline settings.gradle:7]: Wrong module name

**Review by atabekm — CHANGES_REQUESTED** [review]

**Comment by atabekm** [comment]
Recreated as https://github.com/shagalalab/qarejet-android/pull/34, so closing this one

**Pull request closed without merging**